projects/plugins/crm/src/automation/class-base-condition.php

```diff
@@ -8,17 +8,17 @@
 namespace Automattic\Jetpack\CRM\Automation;
 
 /**
- * Base Condition Step
+ * Base Condition Step.
  *
  * @inheritDoc
  */
```

Comment:
We could add a ` * @since $$next-version$$` tag here.

Reply:
I haven't checked docs or code but I think inheritDoc should be wrapped in `{}`: `{@inheritDoc}`

Reply:
Yes, PHPDocumentor 2 accepted it, but the new versions require to be wrapped in `{}` 

Fixed.